.eslintrc.jsx

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
This file can't have `jsx` extension. It can be `.js` or `.json`. Only React components use JSX.